### PR TITLE
Rust 2021

### PR DESCRIPTION
**Pull request by harlantwood**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -11,6 +11,13 @@ jobs:
      - uses: actions/checkout@v2
      - uses: actions-rs/toolchain@v1
        with:
-         toolchain: 1.55
+         toolchain: 1.56
          target: wasm32-unknown-unknown
+
+     - run: rustup update stable
```
[the remark this answers is not in the document]
Comment (harlantwood) [reply .github/workflows/test.yml:17]: At the time of making that PR, that line was needed to get the latest and correct version of cargo. It’s possible that the tool chain action has been updated, in which case this line would not be needed anymore.

**Comment by Connoropolous** [comment]
thanks for this @harlantwood 
just waiting till the right time to give this a trial run

**Comment by Connoropolous** [comment]
@weswalla this could also be merged, as part of your process?

**Comment by weswalla** [comment]
sure

**Pull request merged**